### PR TITLE
Updated class-based meta steps to friendly names

### PR DESCRIPTION
**Pull request by bitwiseman**

### CODE REVIEW

**Comment by bitwiseman** [comment]
@reviewbybees

File: pipeline-examples/ansi-color-build-wrapper/AnsiColorBuildWrapper.groovy [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // This shows a simple build wrapper example, using the AnsiColorBuildWrapper plugin.
 node {
     // This is the current syntax for invoking a build wrapper, naming the class.
```
Comment (daniel-beck) [inline pipeline-examples/ansi-color-build-wrapper/AnsiColorBuildWrapper.groovy]: This comment should be removed then.

File: pipeline-examples/ansi-color-build-wrapper/AnsiColorBuildWrapper.groovy [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // This shows a simple build wrapper example, using the AnsiColorBuildWrapper plugin.
```
Comment (daniel-beck) [inline pipeline-examples/ansi-color-build-wrapper/AnsiColorBuildWrapper.groovy]: `AnsiColorBuildWrapper` is not what the plugin is called.

**Comment by bitwiseman** [comment]
@jglick @daniel-beck Fixed.

**Pull request merged**